src/popup/components/MultisigProposalConsensus.vue

```diff
@@ -174,31 +188,41 @@ export default defineComponent({
       pendingMultisigTxConfirmedBy,
       pendingMultisigTxRefusedBy,
       isPendingMultisigTxCompleted,
+      openModal,
+      MODAL_CONSENSUS_INFO,
     };
   },
 });
 </script>
 
 <style lang="scss" scoped>
-@use '../../styles/variables';
-@use '../../styles/typography';
-@use '../../styles/mixins';
+@use "../../styles/variables";
+@use "../../styles/typography";
+@use "../../styles/mixins";
```

Comment:
I don't see a reason for making this change - we are using single apostrophes everywhere.